tools/build/Makefile.targets

```diff
@@ -202,11 +202,44 @@ endif
 
 PHONY += cheat-sheet
 
-DEFAULT_ALIAS_FILE := $(build_aliasesdir)/50-default.json
-
-ifeq (y,$(USE_FLOW))
-gen-aliases: $(FLOW_NODE_TYPE_ALIAS)
-	$(Q)$(MKDIR) -p $(build_aliasesdir)
-	$(Q)echo "     "GEN"   "$(DEFAULT_ALIAS_FILE)
-	$(Q)$(FLOW_NODE_TYPE_ALIAS) --output=$(DEFAULT_ALIAS_FILE) $(all-gens)
-endif
+ifneq ($(USE_NODEJS),)
```

Comment:
From here on it seems you got your rebase wrong (remove the + lines, please). Testing the rest now, they look good from a 1st view